ruma-appservice-api/src/event/push_events/v1.rs

```diff
@@ -43,3 +42,33 @@ impl Response {
         Self
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::Request;
```

Comment:
Please move this into its own import group after the external crate imports.

Reply:
Changed!